$OpenBSD: patch-lib_rubygems_require_paths_builder_rb,v 1.1 2011/03/22 20:23:10 jeremy Exp $

Create .require_paths file for compatibility with PLISTs for
ruby 1.8 and ruby 1.9.

--- lib/rubygems/require_paths_builder.rb.orig	Tue Mar 15 10:48:59 2011
+++ lib/rubygems/require_paths_builder.rb	Tue Mar 22 13:09:17 2011
@@ -14,5 +14,5 @@ module Gem::RequirePathsBuilder
       file.puts spec.bindir if spec.bindir
     end
   end
-end if Gem::QUICKLOADER_SUCKAGE
+end
 
